### PR TITLE
Update contributing information

### PR DESCRIPTION
**Pull request by jacobmoss-wf**
@Workiva/messaging-pp

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Raven
<details>
<summary>Number of Findings: 0</summary>

</details>

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -30,4 +27,3 @@ Current Frugal Maintainers
  - Charlie Strawn
  - Steven Osborne
  - Tyler Rinnan
- - Tyler Treat
```
Comment (aldenpeterson-wf) [inline CONTRIBUTING.md]: you could make him a maintainer again! hah

Comment (tylertreat) [inline CONTRIBUTING.md]: RIP

**Comment by stevenosborne-wf** [comment]
+1

**Pull request merged**